### DISCUSSION TITLE
Have I understood this JAX model correctly?

### DISCUSSION THREAD

**Discussion by sarakodeiri** [jax-ml/jax#18929] [Q&A]
Hi everyone.
I'm new to JAX and want to see if I've understood a code correctly.

This is the code:
https://github.com/google-deepmind/jax_privacy/blob/main/jax_privacy/experiments/image_classification/models/cifar.py

And this is the schematic of what I've figured out:
<img width="542" alt="WRN" src="https://github.com/google/jax/assets/34372115/52e8fa61-bcb0-4d54-9284-a3aa9d59af1a">

Can anyone please help me verify this?

**Comment by ThatBitBoss** [comment]
Looking at the conversations, I am wondering what you mean by "verify" - what would an answer look like? I ask because I know how frustrating it can be when you ask a question and no answers,
